### PR TITLE
Decorate a class

### PR DESCRIPTION
**Pull request by Atifzada**
### In this PR following changes have been made:

- Added `Nameable` class
- Implemented a method called `correct_name`
- Made sure that `Person` class inherits from the` Nameable`
- Person class has a method `correct_name`
- Created `CapitalizeDecorator` a class that inherits from the `base_decorator` class
- Created `TrimmerDecorator` a class that inherits from the `base_decorator` class
- Implemented a method `correct_name` that made sure that the output of `@nameable.correct_name `has a maximum of 10 characters. If it's longer it should trim the word.

### CODE REVIEW

File: trimmer_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    trimmed_name = @nameable.correct_name[0...10]
```
Comment (VICTOR-CHUKWU) [inline trimmer_decorator.rb]: - [x] According to the project requirement name should be trimmed to a maximum of 10 characters to achieve that you need to   trimmed_name = @nameable.correct_name[0...9], kindly implement that

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Atifzada  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

- [x] your MIT link is broken in your README kindly fix this

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: trimmer_decorator.rb (hunk 1 above)
Comment (Atifzada) [reply trimmer_decorator.rb]: Thank you so much for your valuable review. I will come up with solution

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Copyright 2023, Atif Zada
+
+PORTFOLIO can be a webpage/website, a web app, a software and so on
+Permission is hereby granted, free of charge, to any person obtaining a copy of this PORTFOLIO and associated documentation files, to deal in the PORTFOLIO without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the PORTFOLIO, and to permit persons to whom the PORTFOLIO is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the PORTFOLIO.
+
+THE PORTFOLIO IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE PORTFOLIO OR THE USE OR OTHER DEALINGS IN THE PORTFOLIO.
```
Comment (Ibizugbe) [inline LICENSE.md]: [OPTIONAL] I suggest you kindly personalize this license to suit your project as it currently gives your project a less professional look🥲.

**Review by Ibizugbe — APPROVED** [review]
Hi @Atifzada 👋👋, 

VeryNiceNiceGIF


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] `mit` license link has been fixed👍.
- [x] `correct_name` method has been refactored as requested by the previous reviewer👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: LICENSE.md (hunk 2 above)
Comment (Atifzada) [reply LICENSE.md]: Sure , thank you

Comment (Atifzada) [reply LICENSE.md]: Change the license and personalize it to suit the project

**Pull request merged**